spec/models/friendship_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Friendship, type: :model do
+  context 'Sending a friendship request it most ' do
+    before(:all) do
+      @user = User.new(username: 'example', email: 'example@user.com',
```

Comment:
:+1:  Good job implementing new tests. I would suggest to work also on the controllers so you can test the CRUD operations as well